File: API/Entity/Board.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace API.Entity {
    public class Board {
        [Key]
        [DatabaseGenerated (DatabaseGeneratedOption.None)]
        public Guid Id { get; set; }
        public string Title { get; set; }
        public ICollection<List> Lists { get; set; } = new List<List> ();
    }
}

File: client/src/app/_models/board.ts
export interface board{
    id: string;
    name: string;
    lists: lists[]
}

 interface lists{
    id: string;
    title: string;
    order: number,
    items: items[]

}
 interface items{
    id: string;
    title: string;
    order: number,
  

}
